rasa/nlu/test.py

```diff
@@ -1454,7 +1454,7 @@ def remove_pretrained_extractors(pipeline: List[Component]) -> List[Component]:
     return pipeline
 
 
-def run_evaluation(
+async def run_evaluation(
```

Comment:
It's also called from [here](https://github.com/RasaHQ/rasa/blob/5c1e92d1eaaa005e2a6f56db37abcf3e77110237/rasa/server.py#L988). We need to add an `await` there so that our `run_evaluation` coroutine is actually executed

Reply:
Some tests (e.g. [this](https://github.com/RasaHQ/rasa/blob/0426f1558f49b019d402b082507ec0a0ee7a0ef3/tests/nlu/test_evaluation.py#L361)) also use it. I can highly recommend using Pycharms `Find Usages` to see where a function / method is used in the code.